src/core/QCheck.ml

```diff
@@ -1118,14 +1145,20 @@ let string_gen gen =
 
 let string = string_gen Gen.char
 let string_of_size size = string_gen_of_size size Gen.char
-let small_string = string_gen_of_size Gen.small_nat Gen.char
-
-let printable_string = string_gen Gen.printable
-let printable_string_of_size size = string_gen_of_size size Gen.printable
-let small_printable_string = string_gen_of_size Gen.small_nat Gen.printable
-
-let numeral_string = string_gen Gen.numeral
-let numeral_string_of_size size = string_gen_of_size size Gen.numeral
+let string_small = string_gen_of_size Gen.small_nat Gen.char
```

Comment:
AFAIU, this adds new `arbitrary` `string` aliases without exposing them in the interface `QCheck.mli`:
- `string_small`
- `string_printable`
- `string_printable_of_size`
- `string_small_printable`
- `string_numeral`
- `string_numeral_of_size`

I think they should also be added to the CHANGELOG.

Reply:
It is added, thanks for spotting that :smile:.